src/bindings/python/src/openvino/runtime/opset1/ops.py

```diff
@@ -1792,11 +1791,13 @@ def pad(
 
 @nameable_op
 def parameter(
-    shape: TensorShape, dtype: NumericType = np.float32, name: Optional[str] = None
+    shape: TensorShape, dtype: Union[NumericType, Type] = np.float32, name: Optional[str] = None
 ) -> Parameter:
     """Return an openvino Parameter object."""
-    element_type = get_element_type(dtype)
-    return Parameter(element_type, PartialShape(shape))
+    return Parameter(get_element_type(dtype)
```

Comment:
Does it work for constant op too?

Reply:
I can see update for `make_constant_node` helper, but there is also a separate function to create `constant` op in opset1/ops.py file, which could be updated to `dtype: Union[NumericType, Type]`:

https://github.com/openvinotoolkit/openvino/blob/6b102784d672a9b020745d937146be0a6f658e27/src/bindings/python/src/openvino/runtime/opset1/ops.py#L315-L323

Reply:
Yes it works, just need to update hints for types:)